project/Dependencies.scala

```diff
@@ -4,13 +4,14 @@ import sbt._
  * External libraries used in the project with versions.
  */
 object Dependencies {
-  lazy val sparkVersion: String = sys.props.get("spark.version").getOrElse("3.1.2")
-  lazy val hadoopVersion: String = sys.props.get("hadoop.version").getOrElse("3.2.0")
+  lazy val sparkVersion: String = sys.props.get("spark.version").getOrElse("3.2.1")
+  lazy val hadoopVersion: String = sys.props.get("hadoop.version").getOrElse("3.3.1")
+  lazy val deltaVersion: String = "1.2.0"
```

Comment:
We must update versions here:
https://github.com/osopardo1/qbeast-spark/blob/f26d28ea86d6c45233be7ddb6b8e47bb98ad3c6d/README.md?plain=1#L21-L23

And here:
https://github.com/osopardo1/qbeast-spark/blob/f26d28ea86d6c45233be7ddb6b8e47bb98ad3c6d/README.md?plain=1#L156-L160

Reply:
Thanks, I did not notice! 
But then a question comes to my mind.

Is it best if we change the qbeast-spark version to `0.3.0-alpha` too?

Reply:
Yup, I think we can bump to a `0.3.0-alpha`, as this PR will include some breaking changes